vhdbuilder/packer/init-variables.sh

```diff
@@ -161,6 +161,18 @@ if [[ "$MODE" == "sigMode" || "$MODE" == "gen2Mode" ]]; then
 				--hyper-v-generation ${HYPERV_GENERATION} \
 				--location ${AZURE_LOCATION} \
 				--features SecurityType=ConfidentialVMSupported
+		elif [[ ${OS_VERSION} == "22.04" ]]; then
+			az sig image-definition create \
+				--resource-group ${AZURE_RESOURCE_GROUP_NAME} \
+				--gallery-name ${SIG_GALLERY_NAME} \
+				--gallery-image-definition ${SIG_IMAGE_NAME} \
+				--publisher microsoft-aks \
+				--offer ${SIG_GALLERY_NAME} \
+				--sku ${SIG_IMAGE_NAME} \
+				--os-type ${OS_TYPE} \
+				--hyper-v-generation ${HYPERV_GENERATION} \
+				--location ${AZURE_LOCATION} \
+				--features SecurityType=TrustedLaunch
```

Comment:
Just wondering any reason we need TrustedLaunch for temporary image definition to capture Gen2 22.04 disk? with this option, when the temporary SIG version is converted to VHD blob with SAS in following step, it fails with following error.

Message: The provided gallery image: /subscriptions/8ecadfc9-d1a3-4ea4-b844-0d9f87e4d7c8/resourceGroups/aksvhdtestbuildrg/providers/Microsoft.Compute/galleries/PackerSigGalleryEastUS/images/2204Gen2/versions/1.1658352289.30585 supports creation of OS disk only with SecurityType as TrustedLaunch.

Reply:
its for TL feature support for ACC. we decided to go forward with Vanilla 22.04 and take up TL + 22.04 later.